src/libs/NumberUtils.ts

```diff
@@ -1,11 +1,9 @@
-import _ from 'underscore';
 import CONST from '../CONST';
 
 /**
  * Generates a random positive 64 bit numeric string by randomly generating the left, middle, and right parts and concatenating them. Used to generate client-side ids.
- * @returns {String} string representation of a randomly generated 64 bit signed integer
  */
-function rand64() {
+function rand64(): string {
```

Comment:
```suggestion
/**
 * Generates a random positive 64 bit numeric string by randomly generating the left, middle, and right parts and concatenating them. Used to generate client-side ids.
 * @returns string representation of a randomly generated 64 bit signed integer
 */
function rand64(): string {
```

We should keep the @returns description.

Reply:
Fixed @fabioh8010 :)